salt/preflight/map.jinja

```diff
@@ -0,0 +1,5 @@
+{% import_yaml 'preflight/defaults.yaml' as settings with context %}
+
+{% set defaults = salt['grains.filter_by']({
```

Comment:
You call it defaults here but you import settings in the state would suggest to call the imported yaml `defaults` and this one `settings` :) 

And also as this is the dict for the preflight I suggest to name it `preflight`, because for me it's really settings here it's more "preflight stuff"